huggingface-gpt2/ort_addon/ort_supplement/scripts/run_lm_gpt2.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+num_gpus=4
+use_ort=true
+
+export TRAIN_FILE=/data/wiki.train.tokens
+export TEST_FILE=/data/wiki.test.tokens
+
+RUN_FILE=/workspace/transformers/examples/run_language_modeling_ort.py
+RESULT_DIR=/workspace/results
+
+if [ "$use_ort" = true ] ; then
+    echo "Launching ORT run:"
+    RUN_CMD="mpirun -n ${num_gpus} --allow-run-as-root python $RUN_FILE --ort_trainer --output_dir=$RESULT_DIR/output-ort"
+else
+    echo "Launching PyTorch run:"
+    RUN_CMD="mpirun -n ${num_gpus} --allow-run-as-root python $RUN_FILE --output_dir=$RESULT_DIR/output-pytorch"
```

Comment:
If we dont use ort, it will still run the run_language_modeling_ort.py file?

Reply:
Resolved by adding changes for running with ORT to the existing run_language_modeling.py to avoid duplication of most of the code and enable running with ORT and PyTorch with the same python script.